bundles/org.openhab.binding.bluetooth.radoneye/README.md

```diff
@@ -0,0 +1,47 @@
+# radoneye
+
+This extension adds support for [RadonEye](http://radonftlab.com/radon-sensor-product/radon-detector/rd200/) radon bluetooth detector. 
+
+## Supported Things
+
+Following thing types are supported by this extension:
+
+| Thing Type ID       | Description                            |
+| ------------------- | -------------------------------------- |
+| radoneye            | Original RadonEye  (RD200)             |
+
+## Discovery
+
+As any other Bluetooth device, RadonEye devices are discovered automatically by the corresponding bridge. 
+
+## Thing Configuration
+
+Supported configuration parameters for the things:
+
+| Property                        | Type    | Default | Required | Description                                                     |
+|---------------------------------|---------|---------|----------|-----------------------------------------------------------------|
+| address                         | String  |         | Yes      | Bluetooth address of the device (in format "XX:XX:XX:XX:XX:XX") |
+| refreshInterval                 | Integer | 300     | No       | How often a refresh shall occur in seconds                      |
+
+## Channels
+
+Following channels are supported for `RadonEye` thing:
+
+| Channel ID         | Item Type                | Description                                 |
+| ------------------ | ------------------------ | ------------------------------------------- |
+| radon              | Number:Dimensionless     | The measured radon level                    |
```

Comment:
In the XML, you define the channel as `Number:Density`. Which definition is right?

Reply:
Sorry for the ridicules delay in fixing this. 

I have checked how the Airthings does it, and it is now aligned with that - the number has a dimension and it is density. Also all the time there was a specific SI calculation so it should be all fine.